lifechanger/src/app/models/project-card.ts

```diff
@@ -0,0 +1,11 @@
+export interface ProjectCardsInfo {
+  imageAndAlt: {
```

Comment:
i suggest you to rename it a little bit, imageAndAlt => image, image => urr or path, alt is fine

Reply:
And also you will need to update this naming in all places where you're using model(service, components) 
But it's more optional point, you can not fix it, it just will be better in terms of naming

Reply:
👍Thank you, dear Mentor, for the remark, I repaired:)